dist/datasource.js

```diff
@@ -164,9 +175,24 @@ function (angular, _, dateMath, moment) {
     this._doQuery = function (from, to, granularity, target, scopedVars) {
 
       function splitCardinalityFields(aggregator) {
+
         if (aggregator.type === 'cardinality' && typeof aggregator.fieldNames === 'string') {
           aggregator.fieldNames = aggregator.fieldNames.split(',')
         }
+
+        //adds json type aggregator
+        if(aggregator.type === 'json'){
+          if(aggregator.value.includes('$')){
```

Comment:
should we check this or just always do it ?
cause even `[[domain]]` is valid

Reply:
moved template expander  common so that its not specific with variable